bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
We actually want exceptions enabled when linking statically, because Arbiter relies upon them internally.

The "real" answer here is that Arbiter should be dynamically linked in most cases—I just set up static linking first because it was easier.